### ISSUE TITLE
Can you give the data from 0.00-0.20 results?

### ISSUE THREAD

**Issue by QWTforGithub** [humanpose1/MS-SVConv#10]
Hi,I would like to make some experimental comparisons according to your work. Could you give me the Feat Macthing Ratio from 0.00 to 0.20? The results I run by myself may not meet the requirements of the results you run, so I hope you can give me, thank you.

Looking forward your reply.

**Comment by humanpose1** [comment]
Hi, Sure, but I'm not sure what you exactly want. You want the FMR of 3DMatch for MS-SVConv 3 scales (or ETH) ?
Can you specify the dataset ? and the model ?

**Comment by QWTforGithub** [comment]
> Hi, Sure, but I'm not sure what you exactly want. You want the FMR of 3DMatch for MS-SVConv 3 scales (or ETH) ?
> Can you specify the dataset ? and the model ?

Yes. May I venture to ask, has your article been accepted? If accepted, I need your data on 3DMatch(3 head). Of course, if you can provide 1 head of 3DMatch data that would be much better, I would appreciate it.

**Comment by QWTforGithub** [comment]
> > Hi, Sure, but I'm not sure what you exactly want. You want the FMR of 3DMatch for MS-SVConv 3 scales (or ETH) ?
> > Can you specify the dataset ? and the model ?
> 
> Yes. May I venture to ask, has your article been accepted? If accepted, I need your data on 3DMatch(3 head). Of course, if you can provide 1 head of 3DMatch data that would be much better, I would appreciate it.
By the way, do you know how to get the gradient of MinkowskiEngin in a certain layer? I have been troubled by this problem for a long time. The network built by MinkowskiEngin computs that loss propagates backward without calling the hook function bound to a certain layer, so I don't know how to get the corresponding gradient, any suggestions would be appreciated，thank you.

**Comment by humanpose1** [comment]
For the second question, I'm not sure but have you tried `model.__dict__` ? I got the attribute `'_backward_hooks'`. As for each minkowski convolution, there are an attribute `kernel` and `bias` which are torch Tensor. so it has a grad. I hope it can help you.

**Comment by humanpose1** [comment]
[Here](https://cloud.mines-paristech.fr/index.php/s/etKWyqxjlllpjwi), you have the results of MS-SVConv 3 scales on 3DMatch ([here](https://cloud.mines-paristech.fr/index.php/s/0hFkjjjsMpxQsUk) for MS-SVConv 1 scale ). the feature match ratio for each scene individually and the hit ratio also (after a symmetric test). You have other information as well  (error of translation or error of rotation).
NB: you can open it with pandas
```python
df = pd.read_csv(your_csv_file)
print(df.groupby("name_scene").mean())
```
Does it help you ?

**Comment by QWTforGithub** [comment]
> [Here](https://cloud.mines-paristech.fr/index.php/s/etKWyqxjlllpjwi), you have the results of MS-SVConv 3 scales on 3DMatch ([here](https://cloud.mines-paristech.fr/index.php/s/0hFkjjjsMpxQsUk) for MS-SVConv 1 scale ). the feature match ratio for each scene individually and the hit ratio also (after a symmetric test). You have other information as well (error of translation or error of rotation).
> NB: you can open it with pandas
> 
> ```python
> df = pd.read_csv(your_csv_file)
> print(df.groupby("name_scene").mean())
> ```
> 
> Does it help you ?

Thank you for your thoughtful reply!
